app/src/main/java/com/mailerdaemon/app/clubs/ClubDetailBottomSheet.kt

```diff
@@ -0,0 +1,152 @@
+                view.club_web.setOnClickListener { chromeTab!!.openTab(`object`.getString("web")) }
+            }
+            setposts(page = `object`.getString("fb"), view = view)
+        } catch (jsonException: JSONException) {
+            jsonException.printStackTrace()
+        } catch (jsonException: IOException) {
+            jsonException.printStackTrace()
+        }
+    }
+
+    @SuppressLint("ClickableViewAccessibility")
+    private fun setposts(page: String?, view: View) {
+        if (page!!.isEmpty()) return
+        // webView.getViewTreeObserver().addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
+//        @Override
+//        public void onGlobalLayout() {
+        var site = page
+        if (page.length > 25) site = site.substring(25)
+        //          int x=webView.getMeasuredWidth()-657;
+//          int y=webView.getMeasuredHeight();
+        // webView.getViewTreeObserver().removeOnGlobalLayoutListener(this::onGlobalLayout);
+
+//          String s="<!DOCTYPE html>\n" +
+//                  "<html>\n" +
+//                  "<head>\n" +
+//                  "\t<title></title>\n" +
+//                  "</head>\n" +
+//                  "<body style=\"height: 1000px;text-align: center\">\n" +
+//                  "\t<div id='yeep' style='width: 100%;height:100%;text-align:center' >\n" +
+//                  "\t</div>\n" +
+//                  "</body>\n" +
+//                  "\n" +
+//                  "<script type=\"text/javascript\">\n" +
+//                  "\tvar s = document.getElementById(\"yeep\");\n" +
+//                  "\ts.innerHTML+=\"<iframe src='https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2F"+site+"&tabs=timeline&width=\"+screen.width+\"&height=\"+screen.height+\"&small_header=true&adapt_container_width=true&hide_cover=true&show_facepile=false&appId=384900825472866'  style='border:none;height:100%;width: 100%; ' scrolling='yes' frameborder='0' allowTransparency='true' allow='encrypted-media'></iframe>\";\n" +
+//                  "\t//document.write(s.innerHTML);\n" +
+//                  "</script>\n" +
+//                  "</html>\n";
+        val s = "<iframe src=\"https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2F" +
+            "$site&tabs=timeline&width=350&height=700&small_header=true&adapt_container_width=true&hide_cover=true&" +
+            "show_facepile=false&appId=384900825472866\" width=\"100%\" height=\"100%\" style=\"border:none;" +
+            "overflow:hidden\" scrolling=\"no\" frameborder=\"0\" allowTransparency=\"true\"" +
+            " allow=\"encrypted-media\"></iframe>"
+        view.web.loadHtml(s)
+        // }
+        // });
+        view.web.setOnTouchListener { _: View?, event: MotionEvent ->
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) event.action == ACTION_BUTTON_PRESS else {
+            TODO("VERSION.SDK_INT < M")
```

Comment:
This piece of code was to restrict the user from clicking the webview. See if it works properly after removing the if condition.

Reply:
warning show kr rha hai sir bss